catalog/ui/catalog-ui-search/src/main/webapp/component/search-forms/search-forms.less

```diff
@@ -0,0 +1,32 @@
+@{customElementNamespace}search-form {
+
+    display: inline-block;
+    padding: @minimumSpacing;
+    margin: @minimumSpacing;
+    width: 200px;
+    height: 150px;
+    text-align:center;
```

Comment:
Add `vertical-align: top` to keep the choices aligned correctly.

Reply:
done